packages/core/components/AutoField/fields/DefaultField/index.tsx

```diff
@@ -57,6 +57,7 @@ export const DefaultField = ({
         id={id}
         min={field.type === "number" ? field.min : undefined}
         max={field.type === "number" ? field.max : undefined}
+        placeholder={field.type === "text" || field.type === "number" ? field.placeholder : undefined}
```

Comment:
This line needs formatting with prettier (see failing CI). Please run `npm run format` and commit the results.